types/static-kdtree/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for static-kdtree 1.0
+// Project: https://github.com/mikolalysenko/static-kdtree
+// Definitions by: Zheyang Song <https://github.com/ZheyangSong>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.7
```

Comment:
```suggestion
```
why you would need this? It compiles just fine under current DT TS support window

Reply:
Removed. 👍